test_util/cluster_api.py

```diff
@@ -360,7 +362,7 @@ def deploy_test_app_and_check(self, app, test_uuid):
                 msg += "Detailed explanation of the problem: {2}"
                 raise Exception(msg.format(r.status_code, r.reason, r.text))
 
-            assert r.json() == {'username': 'root'}
+            assert r.json() == {'username': self.mesos_user}
```

Comment:
Oh, good catch; I didnt know that. Fortunately this helper only works on the same test_app which will not change docker user, so easy to cover